src/app/page.tsx

```diff
@@ -65,6 +65,21 @@ export default function Home() {
             {isLoading && <Loader2 className="mr-2 h-4 w-4 animate-spin" />}
             Envoyer
           </Button>
+          {isLoading ? null : (
+            <Button
+              variant="destructive"
+              onClick={() => {
+                socket?.emit(
+                  'quitting',
+                  JSON.stringify({
+                    id: player?.id,
+                  }),
+                );
+              }}
+            >
+              Quitter
+            </Button>
+          )}
```

Comment:
We want to display this button only if the player is in the connected players array

```suggestion
          {players.some((item) => item.id === player?.id && (
            <Button
              variant="destructive"
              onClick={() => {
                socket?.emit(
                  'quitting',
                  JSON.stringify({
                    id: player?.id,
                  }),
                );
              }}
            >
              Quitter
            </Button>
          )}
```